test/built-ins/Array/prototype/Symbol.unscopables/change-array-by-copy.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Igalia. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-array.prototype-@@unscopables
+description: >
+    Initial value of `Symbol.unscopables` property
+info: |
+    22.1.3.32 Array.prototype [ @@unscopables ]
+
+    ...
+    12. Perform ! CreateDataPropertyOrThrow(unscopableList, "toReversed", true).
+    13. Perform ! CreateDataPropertyOrThrow(unscopableList, "toSorted", true).
+    14. Perform ! CreateDataPropertyOrThrow(unscopableList, "toSpliced", true).
+    ...
+includes: [propertyHelper.js]
+features: [Symbol.unscopables, change-array-by-copy]
+---*/
+
+var unscopables = Array.prototype[Symbol.unscopables];
+
+for (unscopable of ["toReversed", "toSorted", "toSpliced"]) {
```

Comment:
```suggestion
for (const unscopable of ["toReversed", "toSorted", "toSpliced"]) {
```

Reply:
we still should avoid for-of usage here, and should copy/paste rather than dynamically loop